pagopa/pn-frontend
# Pn 7098 fix safari auto formatting

## Short description
This PR fixes the annoying Safari on iOS (iPhone, iPad) phone auto formatting.
Please, consider there was previous PR already approved to allow us to test in DEV because the webapps cannot run in local phones at the moment.

## List of changes proposed in this pull request
- Added new component for disabling auto formatting
- Added tests

## How to test
Use Safari in iOS and go in contacts page and try to delete an existing phone number. The popup should be properly show message without HTML tags now.
